app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterConnector.scala

```diff
@@ -93,6 +94,12 @@ class ShutterConnector @Inject()(
         s"$urlEvents?type=${encodeQueryParam(EventType.ShutterStateChange.asString)}&namedFilter=latestByServiceName&data.environment=${encodeQueryParam(env.asString)}&data.shutterType=${encodeQueryParam(st.asString)}")
       .map(_.flatMap(_.toShutterStateChangeEvent))
 
+  def shutterEventsByTimestampDesc(filter: ShutterEventsFilter)(implicit reads: HttpReads[Seq[ShutterEvent]], hc: HeaderCarrier): Future[Seq[ShutterStateChangeEvent]] = {
```

Comment:
The `implicit HttpReads[ShutterEvent]` should be provided by this function (internal import), not a parameter, since it is part of the Api - not flexible.
In fact it is already in scope (line 44)

Reply:
I had reluctantly exposed the ```HttpReads``` in order to put a unit test together for the connector which uses a mock ```Http```.  I have now reworked this using an ```any``` matcher, and so have been able to hide this again.